distribution/docs/src/main/resources/content/_configuring/configuring-from-import-intro-contents.adoc

```diff
@@ -1,14 +1,29 @@
-:title: Configuration Export/Import Intro
+:title: Exporting/Importing Configuration Settings
 :type: configuringImport
 :status: published
 :summary: Export a current ${branding} configuration and use it to restore the system to the same state on a brand new installation.
 :order: 00
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or when adding a second node to a highly available deployment.
+==== Exporting/Importing Configuration Settings
+
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or to add a second node to a highly available deployment.
+
+To export the current configuration settings:
+. Run the `migration:export` command from the ${command-console}.
+. Copy the `exported-${project.version}.zip` file from the `exported` (unless specified otherwise when invoking the `migration:export`) directory underneath `<${branding}_HOME>` to a secure place.
```

Comment:
This wording reads awkwardly to me. I would explain where the file is located, then give the copy instruction. Something like:

A file named `exported-${project.version}.zip` will be placed in `<${branding}_HOME>`, unless another destination was specified when invoking `migration:export`. Copy this file to a secure place.